components/Common/Sidebar/index.tsx

```diff
@@ -0,0 +1,31 @@
+import classNames from 'classnames';
+import { useState, type FC } from 'react';
+
+import type { ActiveItem, SidebarGroupType } from '@/types/sidebar';
+import SideBarGroup from 'components/Common/SidebarGroup';
+
+import styles from './index.module.css';
+
+type SideBarGroupProps = { groups: Array<SidebarGroupType> };
+
+const SideBar: FC<SideBarGroupProps> = ({ groups }) => {
+  const [activeItem, setActiveItem] = useState<ActiveItem>();
+
+  return (
+    <aside
+      className={classNames(styles.sideBar)}
```

Comment:
```suggestion
      className={styles.sideBar}
```

No need to use classNames there